CMSgov/mint-app
# [EASI-4387] Customized homepage

# EASI-4387

## Description


- Adds new routing and homepage for customized settings and order
- Card to link to settings page
- Empty states
- Components preserve settings order
- Unit tests

Going to do a final pass with Cypress after all the tables have been implemented

## How to test this change

- Seed database
- Might need to manually add more than 10 to get pagination to occur if you want to test that as well
- Login and verify that My models table is rendered is base user, All models table rendered if admin
- Click the card to edit setttings
- Reorder, remove all, etc
- Verify the table rendered as expected/Figma
- Tests pass
 

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
